packages/localize-tools/CHANGELOG.md

```diff
@@ -109,11 +121,11 @@ Before:
 class HomePage {
   hello() {
     // msgdesc: Greeting to Earth
-    return msg(html`Hello World`);
+    return msg(html` Hello World `);
```

Comment:
Pretty weird that Prettier (I assume) just inserted some whitespace here. Doesn't really matter, just weird.

Reply:
Yeah, weird. I think I reverted it all now...